Scripts/Services/SpeechToText/V1/SpeechToTextServiceExtension.cs

```diff
@@ -531,6 +558,10 @@ private void SendStart()
                 start["end_of_phrase_silence_time"] = EndOfPhraseSilenceTime;
             if (SplitTranscriptAtPhraseEnd != null)
                 start["split_transcript_at_phrase_end"] = SplitTranscriptAtPhraseEnd;
+            if (SpeechDetectorSensitivity != null)
+                start["speech_detector_sensitivity"] = SpeechDetectorSensitivity;
+            if (BackgroundAudioSuppression != null)
+                start["background_audio_suppression"] = BackgroundAudioSuppression;
             start["processing_metrics"] = ProcessingMetrics;
             start["processing_metrics_interval"] = ProcessingMetricsInterval;
```

Comment:
Do we need a null check for these two? `ProcessingMetrics` and `ProcessingMetricsInterval`